_posts/2015-12-18-fengzhichu-theme.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: post
 title: Fengzhichu-Theme
-date: 2015-09-29 12:12:12
-homepage: https://github.com/fengzhichu/fengzhichu-theme/tree/gh-pages
+date: 2015-12-18 17:10:00
```

Comment:
So sorry for that. 


在 2016年1月2日 03:03:58, Günter Kits (notifications@github.com) 写到:

In _posts/2015-12-18-fengzhichu-theme.markdown:

> @@ -1,8 +1,8 @@
>  ---
>  layout: post
>  title: Fengzhichu-Theme
> -date: 2015-09-29 12:12:12
> -homepage: https://github.com/fengzhichu/fengzhichu-theme/tree/gh-pages
> +date: 2015-12-18 17:10:00
Hey! Please don't change the date because this will make the date in front matter and file name out of sync. Don't rename the file either, just keep the date as it was. Thanks!

—
Reply to this email directly or view it on GitHub.